src/main/kotlin/org/jitsi/nlj/transform/node/Node.kt

```diff
@@ -115,7 +115,10 @@ sealed class Node(
         }
     }
 
+    abstract fun trace(f: () -> Unit)
```

Comment:
We should add some docs here, as I think the reasoning for this would be difficult to infer.